pkg/utils/gnmiutils.go

```diff
@@ -133,22 +138,55 @@ func NewGnmiSetRequest(updates []*gnmi.Update, deletes []*gnmi.Path,
 		})
 	}
 
+	if ext111Strategy != nil {
+		val := make([]byte, 4)
+		binary.LittleEndian.PutUint32(val, uint32(*ext111Strategy))
+		ext := configapi.TransactionStrategy{
+			Synchronicity: configapi.TransactionStrategy_Synchronicity(uint32(*ext111Strategy)),
+		}
+		b, err := ext.Marshal()
```

Comment:
when I send down the value `1`, in `onos-config` I get 
```
extension:{registered_ext:{id:111  msg:"\x08\x01"}}
```
and the transaction is always ASYNC, no matter whether I use 0 or 1. This is with `onos-config` 1.6.6 chart

Reply:
@SeanCondon when I try I see that same value but the transaction is marked as `SYNCHRONOUS` in `onos-config`:
```
onos config get transactionsurl
ID                                           INDEX    STATUS.STATE    TRANSACTIONTYPE    CREATED                                    UPDATED                                    DELETED    USERNAME    TRANSACTIONSTRATEGY.ISOLATION    TRANSACTIONSTRATEGY.SYNCHRONICITY
uuid:9a269e21-9563-4120-8b53-cc7f56c42e63    1        APPLIED         Change             2022-02-11 17:34:30.298043021 +0000 UTC    2022-02-11 17:34:30.481434001 +0000 UTC    <nil>                  DEFAULT                          SYNCHRONOUS
```
I added some logs into `onos-config` to verify what's going on and I see:
```
default/onos-config-78d8b6cfd7-lsb7x[onos-config]: 2022-02-11T17:34:30.295Z     INFO    northbound/gnmi v2/set.go:44    Received gNMI Set Request update: [....] extension:{registered_ext:{id:111 msg:"\x08\x01"}}
default/onos-config-78d8b6cfd7-lsb7x[onos-config]: 2022-02-11T17:34:30.295Z     INFO    northbound/gnmi v2/set.go:47    gNMI Set() called by ' () ()'. Groups []
default/onos-config-78d8b6cfd7-lsb7x[onos-config]: 2022-02-11T17:34:30.295Z     INFO    northbound/gnmi v2/extensions.go:32     TEO found gNMI extension {"ext": {"RegisteredExt":{"id":111,"msg":"CAE="}}}
default/onos-config-78d8b6cfd7-lsb7x[onos-config]: 2022-02-11T17:34:30.295Z     INFO    northbound/gnmi v2/extensions.go:37     TEO transactionStrategy  {"transactionStrategy": {"synchronicity":1}}
```

I did a quick test taking the pieces at the ends and things seems to be working fine for me:
```
func main() {

	val := 1
	var ext111Strategy *int = &val

	ext := configapi.TransactionStrategy{
		Synchronicity: configapi.TransactionStrategy_Synchronicity(uint32(*ext111Strategy)),
	}
	b, err := ext.Marshal()
	if err != nil {
		log.Fatal(err)
	}

	log.Println("Marshalled proto: ", b)

	var transactionStrategy configapi.TransactionStrategy

	if err := proto.Unmarshal(b, &transactionStrategy); err != nil {
		log.Fatal(err)
	}
	log.Println("Unmarshalled proto: ", transactionStrategy.Synchronicity)

}
```
outputs:
```
2022/02/11 10:08:16 Marshalled proto:  [8 1]
2022/02/11 10:08:16 Unmarshalled proto:  SYNCHRONOUS

```